[BUG] upgrading/installing rtorrent

### What happened?

Steps to reproduce:

1. box update
2. box upgrade/install rtorrent
3. build fails, rtorrent gone.

Tried every build (0.9.8, 0.9.7, 0.9.6), they all fail. Had to install repo version to get everything running again.

Expected results:

upgrading rtorrent

### Swizzin commit

0acf022

### What OS are you using?

Ubuntu 22.04 (Jammy)

### What architecture is your OS?

amd64

### Relevant logs and output

```plaintext
/bin/bash ../libtool  --tag=CXX   --mode=link g++  -w -O2 -flto=12     -o rtorrent main.o libsub_root.a ui/libsub_ui.a core/libsub_core.a display/libsub_display.a input/libsub_input.a rpc/libsub_rpc.a utils/libsub_utils.a  -lncursesw -l>
libtool: link: g++ -w -O2 -flto=12 -o rtorrent main.o  libsub_root.a ui/libsub_ui.a core/libsub_core.a display/libsub_display.a input/libsub_input.a rpc/libsub_rpc.a utils/libsub_utils.a -lncursesw -ltinfo -lcppunit -L/usr/local/lib /us>
lto1: fatal error: bytecode stream in file '/usr/local/lib/libcurl.a' generated with LTO version 11.3 instead of the expected 12.0
compilation terminated.
lto-wrapper: fatal error: g++ returned 1 exit status
compilation terminated.
/usr/bin/ld: error: lto-wrapper failed
collect2: error: ld returned 1 exit status
make[3]: *** [Makefile:563: rtorrent] Error 1
make[3]: Leaving directory '/tmp/rtorrent/src'
make[2]: *** [Makefile:634: all-recursive] Error 1
make[2]: Leaving directory '/tmp/rtorrent/src'
make[1]: *** [Makefile:453: all-recursive] Error 1
make[1]: Leaving directory '/tmp/rtorrent'
make: *** [Makefile:385: all] Error 2
^[[2m[Mon Jul 17 22:46:58]^[(B^[[m ^[[31m^[[1mERROR     Something went wrong while making rtorrent^[(B^[[m
^[[2m[Mon Jul 17 22:46:58]^[(B^[[m ^[[31m^[[3m  Please consult the above and/or check the log (less -R +G /root/logs/swizzin.log)^[(B^[[m
```

ar cru libsub_root.a command_download.o command_dynamic.o command_events.o command_file.o command_ip.o command_helpers.o command_groups.o command_local.o command_logging.o command_network.o command_peer.o command_throttle.o command_trac>
ar: `u' modifier ignored since `D' is the default (see `U')
ranlib libsub_root.a
/bin/bash ../libtool  --tag=CXX   --mode=link g++  -w -O2 -flto=12     -o rtorrent main.o libsub_root.a ui/libsub_ui.a core/libsub_core.a display/libsub_display.a input/libsub_input.a rpc/libsub_rpc.a utils/libsub_utils.a  -lncursesw -l>
libtool: link: g++ -w -O2 -flto=12 -o rtorrent main.o  libsub_root.a ui/libsub_ui.a core/libsub_core.a display/libsub_display.a input/libsub_input.a rpc/libsub_rpc.a utils/libsub_utils.a -lncursesw -ltinfo -lcppunit -L/usr/local/lib /us>
/usr/bin/ld: cannot find /usr/local/lib/libcurl.a: No such file or directory
collect2: error: ld returned 1 exit status
make[3]: *** [Makefile:568: rtorrent] Error 1
make[3]: Leaving directory '/tmp/rtorrent/src'
make[2]: *** [Makefile:639: all-recursive] Error 1
make[2]: Leaving directory '/tmp/rtorrent/src'
make[1]: *** [Makefile:458: all-recursive] Error 1
make[1]: Leaving directory '/tmp/rtorrent'
make: *** [Makefile:390: all] Error 2
^[[2m[Wed Jul 19 21:52:02]^[(B^[[m ^[[31m^[[1mERROR     Something went wrong while making rtorrent^[(B^[[m
^[[2m[Wed Jul 19 21:52:02]^[(B^[[m ^[[31m^[[3m  Please consult the above and/or check the log (less -R +G /root/logs/swizzin.log)^[(B^[[m

--- a/sources/functions/curl
+++ b/sources/functions/curl
@@ -3,6 +3,7 @@
 
 configure_curl() {
     apt_install cmake libssl-dev libnghttp2-dev libzstd-dev libldap2-dev libssh2-1-dev libpsl-dev
+    rm /usr/local/lib/libcurl.la
 }
 
 build_cares() {

